azure-pipelines.yml

```diff
@@ -0,0 +1,52 @@
+# DevOps Pipeline definition for the OFS Beta Backend Data Pipeline 
+
+# The code branch that will trigger a build
+trigger:
+- develop
+
+# Build Agent Resources
+pool:
+  vmImage: 'ubuntu-latest'
+strategy:
+  matrix:
+    Python36:
+      python.version: '3.6'
+
+# Build Process Tree
+steps:
+
+# Enforce Python Version (see above)
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '$(python.version)'
+  displayName: 'Use Python $(python.version)'
+
+# Create a Python Virtual Environment and download required dependencies
+- script: |
+    python3.6 -m venv .python_packages
+    source .python_packages/bin/activate
+    python3.6 -m pip install --upgrade pip
+    python3.6 -m pip install -r requirements.txt
+  displayName: 'Install Python Dependencies'
+
+# Execute Python Tests
+#- script: |
+#    python3.6 -m pip install -r requirements.txt
+#    pytest -v
+#  displayName: 'pytest'
```

Comment:
can this be removed?

Reply:
We'll leave it in here for now as we will want to uncomment out when we have added tests